app/templates/components/wds-global-navigation/search-modal.hbs

```diff
@@ -0,0 +1,17 @@
+{{#wds-button click=(action 'openModal' 'search') class='wds-global-navigation__modal-control wds-global-navigation__modal-control-search' text=true type='button'}}
+	{{wds-icon name='magnifying-glass'}}
+{{/wds-button}}
+
+{{#if isOpen}}
+	<div class="wds-global-navigation__modal wds-search-modal">
+		<div class="wds-search-modal__form-and-links-wrapper">
+			{{wds-global-navigation/search
+				model=model
+				onSearchToggleClicked=onSearchToggleClicked
```

Comment:
I'd vote for always using (action onSearchToggleClicked) not much difference but I think it shows clearer that a function is passed